fiftyone/utils/eval/detection.py

```diff
@@ -193,34 +243,31 @@ def evaluate_detections(
     else:
         _samples = samples.select_fields([gt_field, pred_field])
 
+    def _map_fnc(sample) -> List[Any]:
+        return _compute_matches_single(
+            sample,
+            eval_method,
+            eval_key,
+            save=save,
+            processing_frames=processing_frames,
+            tp_field=tp_field,
+            fp_field=fp_field,
+            fn_field=fn_field,
+        )
+
+    if parallelize_method is None and workers is None:
```

Comment:
Maybe need to think a bit more about whether we need to add an explicit "parallel" param or just set workers to 1